### PR TITLE
Cypress/fe 1718 updating the readme.md file for cypress

### PR DESCRIPTION
**Pull request by DlgSHi**
Cypress/fe 1718 updating the readme.md file for cypress

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -7,22 +7,41 @@
 4. Run storybook `npm run storybook`
 5. Open new terminal at the same path
 6. Run cypress `npm run test-cypress`
-7. If you would like to run cypress in command line (headless browser for continous integration) use `npm run test:ci`
-7.1 If you would like to run specific cypress tests in command line (headless browser for continous integration) use:
-  - for Windows `npm run test:ci --spec 'cypress/features/[tests-type]/[featureFileName].feature'`
-  - for MacOS `npm run test:ci -- --spec 'cypress/features/[tests-type]/[featureFileName].feature'`
+7. If you would like to run cypress in command line (headless browser for continous integration) use `npm run test-cypress`
```
Comment (marcinstachowicz) [inline cypress/README.md]: remove this line because `npm run test-cypress` will open full cypress window UI

File: cypress/README.md [hunk 2]
```diff
@@ -7,22 +7,41 @@
...
 
 ## Coding standards
 1. Use ESlint plugin to Visual Studio Code to make sure code format is preserved
 2. Use kebab-case for all `*.js` file names
 3. Use camelCase for `*.feature` file names
+4. Use BDD keywords:
+  * GIVEN - the given part describes the state of the world before you begin the behavior you are specifying in this scenario. You can think of it as the pre-conditions to the test
+  * WHEN - the when section is that behavior that you are specifying. All action you need to perform before making an assertion
+  * THAN - the then section describes the changes you expect due to the specified behavior. Regular assertion
+  * AND - each of the section could have own and state. You can use and to avoid using duplication of the keywords (Given / When / Than)
```
Comment (marcinstachowicz) [inline cypress/README.md]: Then instead of Than

File: cypress/README.md [hunk 3]
```diff
@@ -7,22 +7,41 @@
...
+  * Regression test suite, runs on TeamCity nightly and make all regression tests. To use run: `npm run test-cypress-regression`
 
 ## Coding standards
 1. Use ESlint plugin to Visual Studio Code to make sure code format is preserved
 2. Use kebab-case for all `*.js` file names
 3. Use camelCase for `*.feature` file names
+4. Use BDD keywords:
+  * GIVEN - the given part describes the state of the world before you begin the behavior you are specifying in this scenario. You can think of it as the pre-conditions to the test
+  * WHEN - the when section is that behavior that you are specifying. All action you need to perform before making an assertion
+  * THAN - the then section describes the changes you expect due to the specified behavior. Regular assertion
```
Comment (marcinstachowicz) [inline cypress/README.md]: Then instead of Than

File: cypress/README.md (hunk 1 above)
Comment (DlgSHi) [reply cypress/README.md]: fixed

File: cypress/README.md (hunk 2 above)
Comment (DlgSHi) [reply cypress/README.md]: fixed

File: cypress/README.md (hunk 3 above)
Comment (DlgSHi) [reply cypress/README.md]: fixed

**Pull request merged**